### ISSUE TITLE
How do I access the metrics from a logger/trainer?

### ISSUE THREAD

**Issue by erap129** [PyTorchLightning/pytorch-lightning#2469]
Hello,  
I wanted to ask, after training a model with a trainer object (and using a logger, such as a wandblogger), I want to access the validation loss of the last epoch. It seems simple but I wan't able to find how to do this in any documentation.  
I tried searching the inner parameters of the Logger with python's dir function but got lost there.  
Any help would be appreciated. Thank you very much!  
  
I am on a windows+conda environment if it matters.

**Comment by rohitgr7** [comment]
I am using `trainer.callback_metrics` to access latest metrics.

**Comment by erap129** [comment]
Thanks! I see this only gives metrics for the last epoch. Do you know how to access metrics for all epochs?

**Comment by rohitgr7** [comment]
I asked this on slack forums yesterday, got the response that with Wandb you can use `trainer.logger.experiment.summary`. I haven't checked it, can you try??
